packages/api/src/Path/Path.service.ts

```diff
@@ -36,6 +36,13 @@ export class PathService {
     return userPaths;
   }
 
+  // redo this path query
+  async findUnjoinedByUser(userId: string): Promise<Path[]> {
+    const paths = await this.findAll();
+    const joined = await this.findByUser(userId);
+    return paths.filter(p => !joined.some(j => p.id === j.pathId));
+  }
```

Comment:
Would it be possible to do this in one query somehow? This could get quite expensive later on ;)

Reply:
Okay sure I'll see if I can do that